PocketBook issue

@NiLuJe 
Using https://github.com/koreader/koxtoolchain/commit/ca4481e00fb8b6081c578c0241185444c7ca5ff3
```

Initially reported by: NiLuJe
URL: https://github.com/NiLuJe/crosstool-ng

Comment:
PocketBook
i.MX5 & up. Based on the awfully old original SDK, hence the oldies.
Possibly prefer the new, upstream Clang SDKs if possible:
https://github.com/pocketbook/SDK_6.3.0
it's for i.MX6 & up, a BR GCC 6.3.0 + Clang 7 TC, on glibc 2.23 / Linux 3.0.35

***********************************************************

WARNING! This sample may enable experimental features.
         Please be sure to review the configuration prior
         to building and using your toolchain!
Now, you have been warned!

***********************************************************

Now configured for "arm-pocketbook-linux-gnueabi"
  IN    config.gen/arch.in
  IN    config.gen/kernel.in
  IN    config.gen/cc.in
  IN    config.gen/binutils.in
  IN    config.gen/libc.in
  CONF  config/config.in
make: *** No rule to make target 'upgradeconfig'.  Stop.
The command '/bin/sh -c ./build_pb_tc.sh' returned a non-zero code: 2
make: *** [Makefile:6: build] Error 2
```

--- a/gen-tc.sh
+++ b/gen-tc.sh
@@ -60,7 +60,8 @@ Build_CT-NG() {
 			unset CFLAGS CXXFLAGS LDFLAGS
 			ct-ng "${tc_target}"
 			ct-ng oldconfig
-			ct-ng upgradeconfig
+			# That requires the 1.24 branch, so, don't do it on 1.23
+			[ ! -f "steps.mk" ] && ct-ng upgradeconfig
 			ct-ng updatetools
 			nice ct-ng build
 			echo ""

